src/components/ToolMenu/LayerTree/index.less

```diff
@@ -79,3 +79,8 @@
     }
   }
 }
+
+.ant-tree-checkbox {
```

Comment:
This will override all `ant-tree-checkbox` classes in the application. Is this intended?

Reply:
You are right! Due to the antd change we might want that actually if the checkboxes seem to be dislocated in other components too. But for this case i made the selector more specific.